mesh/ncmesh.cpp

```diff
@@ -5228,43 +5292,56 @@ void NCMesh::GetBoundaryClosure(const Array<int> &bdr_attr_is_ess,
                   bdr_edges.Append(nodes[enode].edge_index);
                }
             }
+
+            // If the face is a slave face, collect any non-ghost master face
```

Comment:
This is the section which handles "hidden" boundary faces, namely a master boundary face coincident to a processor boundary. If all the slave faces neighbor to this face are on another rank, this face might not know it's dof need to be marked essential. The `bdr_faces` variable is used to account for this, collecting up such faces so that their dof can be collected and marked as essential vdofs.

Reply:
Multiple comments in this file mention "any masters." Would it be more accurate to say "its master", since each slave handles its one master? Using the plural suggests multiple masters, which I suppose could mean masters at higher levels in the hierarchy, but that is not the case here.

Reply:
The use of any was being coloured by my thinking of faces as three buckets (slave, master, conformal) during this work rather than their relation between individual faces. I think you're correct that its is more precise here, as there's a specific pair of faces and specific relation between them.